Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -360,7 +361,7 @@ public final class PackageInfoMapper: PackageInfoMapping {
         } else {
             let automaticSchemesOptions: ProjectDescription.Project.Options.AutomaticSchemesOptions
             switch packageType {
-            case .external:
+            case .remote:
```

Comment:
It's amazing we can reuse the previous work of supporting test targets and the `.local` case just falls into that bucket 🙌

Reply:
Thanks to solid foundations 👍